index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Photon Extension Kit</title>
+  <link rel="stylesheet" href="extension.css">
+</head>
+<body>
+  <style>
+    html, body {
+      width: 300px;
+    }
+    .view[hidden] {
+      display: none;
+    }
+  </style>
+
+  <div class="view" id="main">
+    <div class="panel-section-list">
+      <div class="panel-list-item" id="show-tabs">Tabs > </div>
+      <div class="panel-list-item" id="show-text-fields">Text Fiels > </div>
+      <div class="panel-list-item" id="show-buttons">Buttons > </div>
+    </div>
+    <div class="panel-section-separator"></div>
+    <div class="panel-section-list">
+      <div class="panel-list-item" id="show-tabs">Radio Buttons</div>
+      <div class="panel-list-item" id="show-text-fields">Checkboxes</div>
+      <div class="panel-list-item" id="show-buttons">Toggle</div>
+    </div>
+    <div class="panel-section-footer">
```

Comment:
This seems like a lot of `panel-section-…` repetition. Could we make it simpler?